gqueries/modules/energy_mix/supply_chart/factsheet_supply_fossil_other.gql

```diff
@@ -4,7 +4,6 @@
         V(energy_distribution_coal, demand) - V(energy_export_coal_baseload, demand) - V(energy_export_coal_backup,demand),
         V(energy_distribution_lignite, demand) - V(energy_export_lignite_baseload, demand),- V(energy_export_lignite_backup, demand),
         V(energy_distribution_uranium_oxide, demand) - V(energy_export_uranium_oxide, demand),
-        V(energy_distribution_non_biogenic_waste, demand) - V(energy_export_non_biogenic_waste, demand),
         V(energy_import_gasoline, demand) - V(energy_export_gasoline_baseload, demand),- V(energy_export_gasoline_backup, demand),
         V(energy_import_kerosene, demand) - V(energy_export_kerosene_baseload, demand),- V(energy_export_kerosene_backup, demand),
         V(energy_import_diesel, demand) - V(energy_export_diesel_baseload, demand),- V(energy_export_diesel_backup, demand),
```

Comment:
I'm seeing some non energetic demands that are subtracted here, but the new methanol flow at least is missing? In a broader sense: a lot of the queries in this folder contain hardcoded nodes and are therefore out of date. These queries are all related to the factsheet, so perhaps it is better to open an issue for this then to try and solve it. What do you think @kndehaan?

Reply:
Hmm, I agree. There's a plan to update/redesign the entire factsheet at some point, right? Do you know if there's already an issue openend for this? Otherwise I would suggest to pick this up when we redesign the factsheet in one go, rather than making it an issue that is picked up whereas the whole structure of the factsheet could be changed eventually..

Reply:
Yes, there is an issue: https://github.com/quintel/etmodel/issues/4042. I will see if I can update the issue.